### ISSUE TITLE
IE 11: comments pagination is out of view

### ISSUE THREAD

**Issue by afercia** [WordPress/twentytwentyone#665]
**Describe the bug**
In Internet Explorer 11, some pages have a layout that causes the browser's horizontal scrollbar to appear. This is not necessarily a problem if there's no content or interactive elements that are out of the initial view, for example when what is on the right side of the page (out of view) is only a container element.

However, when content or interactive elements require horizontal scrolling to be used, that is an accessibility problem that should be avoided.

**To Reproduce**
- in the WP admin > Discussion Settings, set "Break comments into pages ... " to 5
- use Internet Explorer 11
- go to the front end and see a post with many comments, for example the "Template: Comments" post from the theme unit tests data
- observe the comments pagination below the comments
- observe the links to the next pages are out of the initial view and require horizontal scrolling 
= tested on a Windows laptop with a 15.6 inches display, resolution of 1920 x 1080 pixels

**Expected behavior**
The links to be within the initial view (no horizontal scrolling requierd)

**Screenshots**

![Screenshot (178)](https://user-images.githubusercontent.com/1682452/96978887-25f64c80-151f-11eb-88e5-45b1e3a380c2.png)
